### PR TITLE
Adding release 0.3

### PR DESCRIPTION
**Pull request by adam2392**
PR Description
--------------

Release 0.3.

Merge checklist
---------------

Maintainer, please confirm the following before merging:

- [ ] All comments resolved
- [ ] This is not your own PR
- [ ] All CIs are happy
- [ ] PR title starts with [MRG]
- [ ] [whats_new.rst](https://github.com/mne-tools/mne-connectivity/blob/main/doc/whats_new.rst) is updated
- [ ] PR description includes phrase "closes <#issue-number>"

### CODE REVIEW

File: mne_connectivity/tests/test_connectivity.py [hunk 1]
```diff
@@ -313,7 +313,7 @@ def test_events_handling(conn_func):
 @pytest.mark.parametrize(
     'epochs', [
         _make_test_epochs(),
-        np.random.RandomState(0).random((10, 3, 500))
+        # np.random.RandomState(0).random((10, 3, 500))
```
Comment (adam2392) [inline mne_connectivity/tests/test_connectivity.py:316]:
```suggestion
        np.random.RandomState(0).random((10, 3, 500))
```

**Comment by hoechenberger** [comment]
It seems you didn't tag a release on GH so far?

**Comment by adam2392** [comment]
Done!

**Comment by hoechenberger** [comment]
Nice!

**Pull request merged**